### PR TITLE
Minor fixes in post-install step

### PR DESCRIPTION
**Pull request by chaitanyaenr**
Fixes https://github.com/chaitanyaenr/ocp-automation/issues/19

### CODE REVIEW

File: roles/rhcos-post-install/tasks/main.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@
   register: rhcos_region
 
 - name: Get public subnet name
-  shell: aws ec2 describe-subnets --filters "Name=tag:kubernetes.io/cluster/{{ OPENSHIFT_INSTALL_CLUSTER_NAME }},Values=owned" "Name=cidrBlock,Values=10.0.0.0/20" | grep "Name" | awk '{print $3}'
+  shell: aws ec2 describe-subnets --filters "Name=cidrBlock,Values=10.0.0.0/20" | grep "{{ OPENSHIFT_INSTALL_CLUSTER_NAME }}" | grep "Name" | awk '{print $3}'
```
Comment (akrzos) [inline roles/rhcos-post-install/tasks/main.yml]: What about using the captured cluster name in the 1st task? (rhcos_cluster_name.stdout - note untested)  The grep has pluses and minuses, one bad example might be if people come very close to sharing a cluster name (Ex. akrzos-0 and akrzos-0-1 then the automation with akrzos-0 will capture two results right?)

File: roles/rhcos-post-install/templates/infra-node-machineset.yml.j2 [hunk 2]
```diff
@@ -54,7 +54,7 @@ items:
             - filters:
               - name: tag:Name
                 values:
-                - {{rhcos_cluster_name.stdout}}_worker_sg
+                - {{rhcos_cluster_name.stdout}}-worker-sg
```
Comment (akrzos) [inline roles/rhcos-post-install/templates/infra-node-machineset.yml.j2:57]: In the last cluser I deployed I still see the prior format

Ex:
```
            securityGroups:
            - filters:
              - name: tag:Name
                values:
                - akrzosrhperfocpauto0-st6dn_worker_sg
```

This was with build 2019-02-24-045124 so maybe you ran a newer build to see this change?

Comment (chaitanyaenr) [reply roles/rhcos-post-install/templates/infra-node-machineset.yml.j2:57]: I saw the change when using the same release. The pbench and infra nodes are using the default security group as it's not able to find the security group created for workers. About the output you pasted, is it from the machinesets we create or the worker node?

Comment (akrzos) [reply roles/rhcos-post-install/templates/infra-node-machineset.yml.j2:57]: Yep your right, I accidentally grabbed the snippet from the infra nodes when I dumped the machineset yaml... so disregard my comment here.

**Review by akrzos — APPROVED** [review]
LGTM

File: roles/rhcos-post-install/tasks/main.yml (hunk 1 above)
Comment (chaitanyaenr) [reply roles/rhcos-post-install/tasks/main.yml]: Makes sense to use rhcos_cluster_name.stdout instead of grep, update the PR @akrzos.

**Pull request merged**